src/blog/v8-release-81.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'V8 release v8.1'
+author: 'Dominik Inführ'
+avatars:
+  - 'dinfuehr'
```

Comment:
I don't see the referenced avatar files in the PR. Seems like they were omitted from `git add`?

Reply:
Is there a script I need to run for this?

Reply:
No, you need to manually add images with given name to [src/img/_avatars](https://github.com/v8/v8.dev/tree/master/src/_img/avatars) folder.

These should be 2 variants of your avatar with same format, dimensions and comparable file sizes as other avatars in that folder (check them as a guide). You can use [Squoosh.app](https://squoosh.app/) for image compression.